crates/common/src/receipt.rs

```diff
@@ -0,0 +1,60 @@
+use crate::prelude::*;
+
+#[derive(Clone, Default, Debug, PartialEq)]
+pub struct Receipt {
+    pub actual_fee: Option<Fee>,
+    pub events: Vec<crate::event::Event>,
+    pub execution_resources: Option<ExecutionResources>,
+    pub l1_to_l2_consumed_message: Option<L1ToL2Message>,
```

Comment:
I think this field is redundant with the respective transaction which should be of type L1_HANDLER.
```suggestion
```

Reply:
For this reason this field is not present in the rpc.

Reply:
Yep, this seems redundant with the data we already have in the transaction.